doc/source/tokudb/tokudb_background_analyze_table.rst

```diff
@@ -0,0 +1,146 @@
+.. _tokudb_background_analyze_table:
+
+===============================
+TokuDB Background ANALYZE TABLE
+===============================
+
+Prior to :rn:`5.6.27-76.0` release there was no mechanism to optionally and automatically trigger a background analysis or gathering of cardinality statistics on a |TokuDB| tables. These statistics are used for ``JOIN`` optimizations and helping the |MySQL| optimizer choose the appropriate index for a query. If a table statistics or index cardinality becomes outdated, you might see queries which previously performed well suddenly running much slower until statistics are updated again. For |TokuDB| cardinality statistics were only updated when an explicit ``ANALYZE TABLE`` was issued.
+
+Although this operation was an online in the sense that ``ANALYZE TABLE`` within |TokuDB| didn't explicitly lock anything |MySQL| itself would place a read lock on a table being analyzed, preventing writes while the analysis is in progress.
+
+There was also no throttling or scope limiting mechanism for |TokuDB| ``ANALYZE TABLE`` other than time limit (:variable:`tokudb_analyze_time` 5 second default) that it spends on each index in a table.
+
+In |Percona Server| :rn:`5.6.27-76.0` new behavior and variables have been implemented to have the tables automatically analyzed in the background based on a measured change in data. This has been done by implementing the new background job manager that can perform operations on a background thread. 
+
+.. note:: 
+
+  If you're upgrading an existing |TokuDB| installation prior to :rn:`5.6.27-76.0`, the |TokuDB| plugin must be disabled and re-enabled via :program:`ps_tokudb_admin` to register this new plugin. Alternately the plugin can be enabled by running 
```

Comment:
As noted above, only the I_S.tokudb_background_job_status table is a plugin in the MySQL sense, the rest of the functionality exists as part of TokuDB itself.